src/FakeItEasy/Analysis/MustUseReturnValueAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace FakeItEasy.Analysis
+{
+    using System;
+
+    /// <summary>
+    /// Indicates that the return value of a method should not be ignored.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public sealed class MustUseReturnValueAttribute : Attribute

```

Comment:
Actually, I don't even see how you're using it now, except by reflection in `UnusedReturnValueAnalyzer.GetDiagnosticName`.